src/modules/zerion-api/requests/wallet-get-portfolio.ts

```diff
@@ -46,12 +48,14 @@ export async function walletGetPortfolio(
   params: Params,
   options: ClientOptions = CLIENT_DEFAULTS
 ) {
-  const firstAddress = params.addresses[0];
+  const addresses = params.addresses.filter((addr) => isEthereumAddress(addr));
+  invariant(addresses.length > 0, 'Addresses param is empty');
```

Comment:
Is it ok to throw exception for wallets with only solana addresses?

Reply:
This is temporary until backend supports solana
I will add a comment, thanks!